scripts/kube-init.sh

```diff
@@ -30,6 +30,11 @@ trap "clean_exit" EXIT
 # Switch off SE-Linux
 setenforce 0
 
+# Mount root to fix dns issues
+# Define $HOME since somehow this is not defined
+export HOME=/home/travis
```

Comment:
This is unexpected. The [travis doc](https://docs.travis-ci.com/user/environment-variables#default-environment-variables) mentioned that default the environment variables are available and `HOME=/home/travis (do not depend on this value)`. 

If the environment variable is truly missed, I'm wondering if [`~` like the previous implementation](https://github.com/kubernetes-client/python/blob/529c0d3c2fbceca0f38d1cd3ae2eb3d5cd660a07/scripts/kube-init.sh#L76-L77) works for you?

Reply:
I think $HOME works in 14.04, but somehow it’s not there. I was super surprised as well. Yes, ~ works. You cannot see it since I squashed the commits with push -f...